languages/csharp/exercises/concept/classes/.docs/after.md

```diff
@@ -0,0 +1,97 @@
+The primary object-oriented construct in C# is the _class_, which is a grouping of related data ([_fields_][fields]) and behavior ([_methods_][methods]). The fields and methods of a class are known as its _members_.
+
+Access to members can be restricted through access modifiers, the two most common ones being:
+
+- [`public`][public]: the member can be accessed by any code (no restrictions).
+- [`private`][private]: the member can only be accessed by code in the same class.
+
+It is customary to specify an access modifier for all members. If no access modifier is specified, it will default to `private`.
+
+The above-mentioned grouping of related data and behavior plus restricting access to members is known as _encapsulation_, which is one of the core object-oriented concepts.
```

Comment:
If thi is a core concept, should it be in `introduction` rather than  `after`?

Reply:
That was Erik's instinct to. It started there and I asked for it to move out :) my reasoning I don't think teaching the word "encapsulation" is core. The grouping of methods and behaviour is the actual core thing and that's covered in the intro. As you can comfortably solve the exercise without reading this paragraph I think it belongs in after.

Reply:
Yeah. I agree with that insight having thought on it. Students don't need to know this to be able to solve the exercise.